frontend/src/metabase/ui/components/theme/ThemeProvider/ThemeProvider.tsx

```diff
@@ -71,15 +67,16 @@ export const ThemeProvider = (props: ThemeProviderProps) => {
     } as MantineTheme;
   }, [props.theme]);
 
+  const { withCssVariables } = useContext(themeProviderContext);
+
   return (
     <MantineProvider
       theme={theme}
       getStyleNonce={() => window.MetabaseNonce ?? "metabase"}
       classNamesPrefix="mb-mantine"
       // This slows down unit tests like crazy
       cssVariablesSelector={isEmbeddingSdk() ? ".mb-wrapper" : undefined}
-      withCssVariables={!isJest}
-      {...props.mantineProviderProps}
+      withCssVariables={withCssVariables}
```

Comment:
Prevent SDK tests from having CSS variables injected which causes some tests to timeout on CI.